src/ConstructiveSolidGeometry/plotting/CSG/CSG.jl

```diff
@@ -25,7 +25,7 @@ end
                     fillalpha --> 0.2
                     if haskey(plotattributes, :seriestype) 
                         if plotattributes[:seriestype] in [:csg, :wireframe] 
-                            linewidth := 0.5
+                            linewidth := 1
```

Comment:
Do we actually want to force the linewidth here? Why not `-->`?

Reply:
I think it's the best option because if the user changes the linewidth, all linewidths would change if we use -->. As written a user set linewidth will only change the linewidth of positive geometries and will force negative geometries to `linewidth := 1`.